net-wireless/wpa_supplicant/files/wpa_supplicant-2.11-Revert-Mark-authorization-completed-on-driver-indica.patch

```diff
@@ -0,0 +1,50 @@
+From 071336247683d82a74f3567abf67a0b37db856ae Mon Sep 17 00:00:00 2001
```

Comment:
Please include a link to the Gentoo bug at the top and any other references (e.g. the RH bugzilla).

Reply:
I just realized i added the wrong patch. That patch is for something else.

Reply:
Cancel that. It is the right patch. I added references to the Gentoo and RH Bugzillsa bugs.